libraries/ktx/src/khronos/KHR.h

```diff
@@ -11,6 +11,7 @@
 #define khronos_khr_hpp
 
 #include <unordered_map>
+#include <stdexcept>
```

Comment:
This seems unnecessary. I think this is probably a fix for a separate issue (likely a compiler being more strict than before) and should go in its own PR.

Reply:
I found that the include of stdexcept was needed for compiling the project on Windows10 with Visual Studio 2019 and using the cmake that is included in VS. It obviously is not needed under Linux.